charts/quickwit/values.yaml

```diff
@@ -120,21 +137,105 @@ metastore:
     #   cpu: 100m
     #   memory: 128Mi
 
-  strategy:
-    type: RollingUpdate
-    # rollingUpdate:
-    #   maxUnavailable: 5%
-    #   maxSurge: 25%
+  ## Pod distruption budget
+  podDisruptionBudget: {}
+    # maxUnavailable: 1
+    # minAvailable: 2
+
+  persistentVolume:
+    enabled: false
+    # storage: "1Gi"
+    # storageClass: ""
+
+  updateStrategy: {}
+    # type: RollingUpdate
 
   livenessProbe:
     httpGet:
       path: /
-      port: http
+      port: rest
 
   readinessProbe:
     httpGet:
       path: /
-      port: http
+      port: rest
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+
+janitor:
+  # Enable Janitor service
+  enabled: false
+
+  replicaCount: 1
```

Comment:
I don't like having a replicaCount variable for the janitor as it make the user believes we can replicate the janitor.
But this is not true, currently, Quickwit should only run one janitor otherwise it will duplicate some operations, it will not make quickwit go nuts but it will make quickwit slower by making useless metastore queries.

Reply:
Fair enough, removed from values and hardcoded in deployment template then 👍